package.json

```diff
@@ -5,7 +5,6 @@
   "main": "./src/index.js",
   "scripts": {
     "start": "node ./src/index.js",
-    "postinstall": "node ./src/util/deploySlashCommands.js"
```

Comment:
Removed this, because if I add more commands, but I don't run `npm install` or `yarn` they will **not** be registered.